Makefile.calicoctl

```diff
@@ -7,6 +7,12 @@
 # Determine which OS / ARCH.
 OS := $(shell uname -s | tr A-Z a-z)
 ARCH := amd64
+###############################################################################
+# Subcomponent versions:
+GO_BUILD_VER:=latest
+# Set libcalico-go version in glide.yaml first then do a `glide up -v` to update glide.lock.
+LIBCALICOGO_VER := v1.0.0
```

Comment:
This is only for printing the value. `RELEASING.md` mentions what needs to be done before updating the Makefile. But, I see your point. I have updated the comment to be more explicit.